### DISCUSSION TITLE
justify-center class doesn't work

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#8642] [Help] [answered]
I tried to display some cards using data from an api, but when a card with long title on the same row with a card that has a short title, it looks strange
![image](https://user-images.githubusercontent.com/61914365/173715900-e0bdd892-993e-4218-82e7-16fcd62f2c9e.png)
so I tried to use `justify-center` class to center the content of the short card so it wouldn't have too much whitespace on the bottom, but it didn't work out, here is my code:
```jsx
const MovieCard = ({ movie }) => {
  return (
    <Link href={`/movie/${movie.id}`} passHref>
      <div className="bg-white shadow-sm rounded-md cursor-pointer">
        <Image
          src={`https://image.tmdb.org/t/p/w500${movie.poster_path}`}
          width={700}
          height={800}
          className="rounded-t-md"
          alt={movie.title}
        />
        <div className="px-6 py-2 flex flex-col justify-center">
          <h3 className="font-bold text-xl mb-1">{movie.title}</h3>
          <p className="text-gray-700 text-base ">{movie.release_date}</p>
        </div>
      </div>
    </Link>
  );
};
```

**Comment by rohmann** [comment, marked as answer]
Hi @Chandler-Zhu 

I think the issue here is that even though it's "centered" it's only using the minimum amount of space needed so nothing changes visibly. What you want is that text area to grow and fill the rest of the container. You can do this by.

1. Add `flex flex-col` on the card div (the one with the shadow).
2. Add `flex-grow` to the same div where you have `justify-center` already.

You should see the centering kick in at that point. Here's an example: https://play.tailwindcss.com/VDfLT4e3j5
